### PR TITLE
Added article about ELK and monolog.

### PR DESCRIPTION
**Pull request by racinmat**
Another article from the ELK series. Now finally with some PHP.

### CODE REVIEW

File: source/_posts/2017/2017-10-15-how-to-use-ELK-stack.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ perex: '''
 '''
 author: 15
 lang: en
+related_posts: [42]
```
Comment (TomasVotruba) [inline source/_posts/2017/2017-10-15-how-to-use-ELK-stack.md]: Here should be 43

File: source/_posts/2017/2017-10-16-connecting-Monolog-with-ELK.md [hunk 2]
```diff
@@ -0,0 +1,414 @@
+---
+id: 43
+layout: post
+title: "How to connect ELK with monolog"
```
Comment (TomasVotruba) [inline source/_posts/2017/2017-10-16-connecting-Monolog-with-ELK.md]: Uppercase titles shoudl be used here as well

**Comment by TomasVotruba** [comment]
Thanks for the PR

File: source/_posts/2017/2017-10-15-how-to-use-ELK-stack.md (hunk 1 above)
Comment (racinmat) [reply source/_posts/2017/2017-10-15-how-to-use-ELK-stack.md]: fixed

File: source/_posts/2017/2017-10-16-connecting-Monolog-with-ELK.md (hunk 2 above)
Comment (racinmat) [reply source/_posts/2017/2017-10-16-connecting-Monolog-with-ELK.md]: Oh, yeah, stil forgetting this one.

**Comment by TomasVotruba** [comment]
Ahh, autocorrect 🚀 

Merged, thank you

**Pull request merged**